tests/NPSP.py

```diff
@@ -307,6 +311,35 @@ def fill_address_form(self, **kwargs):
             else:
                 locator = locator+"input"
                 self.selenium.get_webelement(locator).send_keys(value)
+                
+    def fill_bge_form(self, **kwargs):
+        """Validates if the affiliated contacts have the added contact details enter Y for positive case and N for negative case""" 
```

Comment:
This docstring looks like it is for a different keyword.

Reply:
removed it